osbs/api.py

```diff
@@ -10,11 +10,25 @@
 from osbs.build import BuildManager, BuildResponse
 from osbs.constants import DEFAULT_NAMESPACE
 from osbs.core import Openshift
-from osbs.exceptions import OsbsResponseException
+from osbs.exceptions import OsbsException, OsbsResponseException
 
 
+def osbsapi(func):
+    def catch_exceptions(*args, **kwargs):
```

Comment:
could you please decorate this with [`wraps`](https://docs.python.org/2/library/functools.html#functools.wraps)? Creating decorators without `wraps` [changes `__doc__` and `__name__`](http://www.jeffknupp.com/blog/2013/11/29/improve-your-python-decorators-explained/).